bin/PrintTokenGraph/Main.cpp

```diff
@@ -28,8 +28,9 @@
 #include <string>
 #include <unordered_set>
 
-template <typename TokT>
-static std::string TokData(const TokT &tok) {
+static bool gShowPrinted = false;
```

Comment:
Do not use mutable global state.

Reply:
I am going to leave this for another PR. I understand this is not an ideal situation, but my goal was to add *some* documentation of what's there, not make what's there substantially better.